README.rst

```diff
@@ -14,11 +14,12 @@ If you will be modifying edx-platform code, the `Open edX Developer Stack`_ is
 a Docker-based development environment.
 
 If you want to run your own Open edX server and have the technical skills to do
-so, `Open edX Ubuntu 16.04 64-bit Installation`_ has instructions to install
-it on an existing Ubuntu 16.04 server.
+so, the `native installation guide`_ has instructions to install
+the latest release, `Koa`_, on an existing Ubuntu 20.04 64-bit server.
 
 .. _Open edX Developer Stack: https://github.com/edx/devstack
-.. _Open edX Ubuntu 16.04 64-bit Installation: https://openedx.atlassian.net/wiki/display/OpenOPS/Native+Open+edX+Ubuntu+16.04+64+bit+Installation
+.. _native installation guide: https://openedx.atlassian.net/wiki/display/OpenOPS/Koa+Native+Open+edX+platform+Ubuntu+20.04+64+bit+Installation
+.. _Koa: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/platform_releases/koa.html
```

Comment:
I would rather keep all the specifics out of this file, so we don't have to touch them in the future.
How about this:
```
If you want to run your own Open edX server and have the technical skills to do
so, `Open edX Installation Options`_ explains your options.

.. _Open edX Installation Options:  https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/60227779/Open+edX+Installation+Options
```

Reply:
Sounds good to me.